app/components/osf-footer/component.ts

```diff
@@ -0,0 +1,12 @@
+import Component from '@ember/component';
+import { serviceLinks } from 'ember-osf-web/const/service-links';
+
+export default class OsfFooter extends Component {
+    supportEmail = 'placeholder';
+    serviceLinks = serviceLinks;
+
+    didRender() {
+        this._super(...arguments);
+        this.set('supportEmail', atob('bWFpbHRvOmNvbnRhY3RAb3NmLmlv'));
```

Comment:
This should come from the config. It's already there by virtue of the support page.

https://github.com/CenterForOpenScience/ember-osf-web/blob/d573bccfce5fd3fb7f1e577fead6ea071d9cdbac/app/support/controller.ts#L10
https://github.com/CenterForOpenScience/ember-osf-web/blob/d573bccfce5fd3fb7f1e577fead6ea071d9cdbac/config/environment.js#L128

If you want to move that key around in config/environment, then we'll have to account for that in the support page as well.